### ISSUE TITLE
[Feature]

### ISSUE THREAD

**Issue by epnixsite** [rspamd/rspamd#3164]
Hey as I understood rspamd failing to update itself if is running behind corporate proxy, can this  please be allowed/functionality added? (Or at least instructions given how to set proxy params)
I seen there is more complains, requests like this from other people, users..

Currently following happening:
error reading https://maps.rspamd.com/rspamd/mid.inc.zst(88.99.142.95:443): connection with http server terminated incorrectly: ssl connect error: syscall fail: Operation timed out

**Comment by vstakhov** [comment]
It is not Rspamd task. Use forward proxy for these purposes, e.g. nginx.

**Comment by epnixsite** [comment]
Your response nonsense here.. Sorry.. RSPAMD should be able to connect to outside world via proxy..  at least should be capable to do so.. as spamassasin can.

**Comment by vstakhov** [comment]
Then use spamassassin, what's the problem?

**Comment by epnixsite** [comment]
I am running rspamd as container.. You Russians seriously need to learn some work ethic..

**Comment by vstakhov** [comment]
Ok, let me rephrase: I do not have enough time for this feature as there are higher priority tasks in my work queue. Thank you for your report!

**Comment by epnixsite** [comment]
That's better answer/explanation. Thanks - I and completely understand. But I think this still shouldn't be closed and etc. as every prod environment would live behind proxy. So proxy support is required. RSPAMD in overal very good- and this is the only feature I am missing so far.

**Comment by vstakhov** [comment]
As possible workaround, you can download these maps manually (e.g. by using cron + wget) and override the default maps paths from external HTTP resources to these local files.

**Comment by epnixsite** [comment]
Thank you!

**Comment by lap-ochka** [comment]
I see no problem here. Vsevolod clearly shows no interest in implementing this feature and as an opensource project owner he's got all the rights to do so. 

In the same time, you can apply for paid feature request either on freelance exchange or here. For example I could add this functionality.

**Comment by epnixsite** [comment]
Lapocka - I am coder/developer same as you guys, I can implement myself if want to.. It's open source project hence - I am adding comments/feature requests.. I only suggesting to make your project better.. Not all about money in open source.. But perhaps in Russia is :)

**Comment by lap-ochka** [comment]
I'm sorry, but I see no point here. 

Either you have a problem and want it to be solved (quick way, suggested above) or this leads nowhere, for Vsevolod already stated he's not been interested in implementing this feature in near future. We can overflow his "issues" with what-seems-to-us-to-as-smart suggestions, but what's the point?

Yeah, opensource is not about money. It's about contribution. And you choose your level of contribution yourself (money, bugfixes, etc).

**Comment by epnixsite** [comment]
I am not sure why you got here involved.. as I already responded to author - I said thank you.. once he listed options.. I guess this is related to mentality and money..

**Comment by lap-ochka** [comment]
> I guess this is related to mentality and money

I guess one day you'll learn not to judge people by the colour of their skin, language or nationality.

**Comment by epnixsite** [comment]
1) read conversation from start
2) read again and try understand
3) stop demonstrating Putin's type mentality.. by trying to move things around..

**Comment by lap-ochka** [comment]
Please don't project your lack of wit and inability to maintain a polite conversation on to other person.

As well as your nazi manners.

**Comment by epnixsite** [comment]
Da pashol ti :) znajish gdie? Occupants

**Comment by lap-ochka** [comment]
The only thing occupied here is common sense by your irrelevant and nationalistic mottos. In the same time, if it's the only thing that cheers you up and helps you sleep at night - please accept my condolences and be yourself, even if it means to be a shauvinistic nationalist pig. Cheers, my friend.

**Comment by epnixsite** [comment]
Dude, I even don't know nor was willing to talk to you.. hence I am referring to mentality of yours.. You kind of trying to show something.. but just talking rubbish here.. All post was finished by my "thank you" message to author an hour ago.. You seems still on FIRE to talk some nonsense here.. Chill and eat something or have a cigarette - as looks like u need one..

**Comment by lap-ochka** [comment]
> All post was finished by my "thank you"

Nope, they were not.
In case your short-term memory is shorter then expected:
"You Russians seriously need to learn ... "
"Not all about money in open source.. But perhaps in Russia is :)"
"I guess this is related to mentality and money.."
"Da pashol ti :)"

Stop being a clown and get a life.
Cheers.
